app/components/quiz/QuizAnswer.js

```diff
@@ -9,22 +11,39 @@ export default React.createClass({
     isSelected: React.PropTypes.bool.isRequired,
     id: React.PropTypes.number.isRequired,
     colors: React.PropTypes.string.isRequired,
-    answerClicked: React.PropTypes.func.isRequired,
+    answerProcessed: React.PropTypes.func.isRequired,
     type: React.PropTypes.string.isRequired
   },
 
   getInitialState: function() {
-    return {color: ''}
+    if (this.props.type !== 'datetime') {
```

Comment:
Checking type feels sort of janky here, especially since the `render` function below is just split into two halves... in the future it might be cleaner to make a bunch of different internal <QuizAnswer> components, which QuizAnswer just renders based on type, but this is fine for now since that approach isn't that much cleaner anyhow...